packages/x-components/src/x-modules/next-queries/components/next-queries-list.vue

```diff
@@ -17,184 +17,219 @@
 </template>
 
 <script lang="ts">
-  import Vue from 'vue';
+  import { computed, ComputedRef, defineComponent, inject, provide, Ref } from 'vue';
   import { NextQuery } from '@empathyco/x-types';
-  import { mixins } from 'vue-class-component';
-  import { Component, Prop } from 'vue-property-decorator';
-  import { Getter } from '../../../components/decorators/store.decorators';
   import { NoElement } from '../../../components/no-element';
-  import { ItemsListInjectionMixin } from '../../../components/items-list-injection.mixin';
   import ItemsList from '../../../components/items-list.vue';
-  import { xComponentMixin } from '../../../components/x-component.mixin';
   import { groupItemsBy } from '../../../utils/array';
   import { ListItem } from '../../../utils/types';
   import { NextQueriesGroup } from '../types';
   import { nextQueriesXModule } from '../x-module';
-  import { XInject } from '../../../components/decorators/injection.decorators';
-  import { HAS_MORE_ITEMS_KEY, QUERY_KEY } from '../../../components/decorators/injection.consts';
+  import {
+    HAS_MORE_ITEMS_KEY,
+    LIST_ITEMS_KEY,
+    QUERY_KEY
+  } from '../../../components/decorators/injection.consts';
+  import { AnimationProp } from '../../../types/index';
+  import { use$x, useGetter, useRegisterXModule } from '../../../composables/index';
 
   /**
    * Component that inserts groups of next queries in different positions of the injected search
    * items list, based on the provided configuration.
    *
    * @public
    */
-  @Component({
+  export default defineComponent({
+    name: 'NextQueriesList',
     components: {
       NoElement,
```

Comment:
Yes, I have replaced the `NoElement` with a `div` element in the template. It could be a possible solution for the moment. In the future, we can discuss a better solution or a way to improve this.